pipeline/installs/multi-user/pipelines-profile-controller/composite-controller.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: metacontroller.k8s.io/v1alpha1
+kind: CompositeController
+metadata:
+  name: profile-controller
+spec:
+  generateSelector: true
+  resyncPeriodSeconds: 10
+  parentResource:
+    apiVersion: v1
+    resource: namespaces
+  childResources:
+  - apiVersion: v1
+    resource: secrets
+    updateStrategy:
+      method: OnDelete
+  - apiVersion: v1
+    resource: configmaps
+    updateStrategy:
+      method: OnDelete
+  - apiVersion: apps/v1
+    resource: deployments
+    updateStrategy:
+      method: InPlace
+  - apiVersion: v1
+    resource: services
+    updateStrategy:
+      method: InPlace
+  - apiVersion: networking.istio.io/v1alpha3
+    resource: destinationrules
+    updateStrategy:
+      method: InPlace
+  - apiVersion: rbac.istio.io/v1alpha1
+    resource: serviceroles
+    updateStrategy:
+      method: InPlace
+  - apiVersion: rbac.istio.io/v1alpha1
+    resource: servicerolebindings
+    updateStrategy:
+      method: InPlace
+  hooks:
+    sync:
+      webhook:
+        url: http://kubeflow-pipelines-profile-controller/sync
```

Comment:
@Bobgy This will likely fail on other platforms or on-prem installations, I had to point to the internal service name: 

```url: http://profile-controller.kubeflow.svc.cluster.local:80/sync  ```

Perhaps this can be configurable via Kustomize vars.

Reply:
Thanks.

is using prefix a GCP specific thing?
I thought it's kubernetes standard behavior.

let me investigate